src/utils/components/CloneVMModal/CloneVMModal.tsx

```diff
@@ -41,6 +41,13 @@ const CloneVMModal: FC<CloneVMModalProps> = ({ headerText, isOpen, onClose, sour
     ),
   );
 
+  const vmUseRunning = useMemo(
+    () =>
+      (source as V1VirtualMachine)?.spec?.running !== undefined &&
+      (source as V1VirtualMachine)?.spec?.running !== null,
```

Comment:
What about when running can be `false`? there are also cases where someone would like to pass a different runStrategy the `Always`?

Reply:
If r`spec.running` is false, it means that the user used this attribute to run/stop the VM so we'll use the same s[attribute changing the `spec.running` . If it's not in the vm spec it means that we'll use `runStrategy` 

I can at this point call the vm API action to run the vm, so the backend will choose what strategy use.

Reply:
runStrategies that start vms are `Always` and `RerunOnFailure`